### PR TITLE
Feature/jkw content delivery endpoints

### LINKED ISSUE #46: Hook up to BE endpoint to get video
### Feature Categories:

- React

**Describe the feature/Bug your gonna work on**

- For ContentDelivery component, connect to BE endpoint to get random video with selected break duration & category

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- This PR covers our MVP functionality! Now, we can fetch random content as a function of the users category (button click) and break interval (settings page dropdown).
     - Creates a `getRandomContent` network request & connects this to `ContentSelection` buttons
     - Adds a `rootUrl` and updates this in existing network requests
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Ran `npm test` in the terminal
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

- Closes #46

### CODE REVIEW

File: src/Components/ContentSelection/ContentSelection.js [hunk 1]
```diff
@@ -2,19 +2,31 @@ import React, { useContext} from 'react'
...
-    //may need a loading screen while this is happening? 
-    setView('content-delivery')
+    const duration = `${settings.breakInterval}:00`
+    
+    getRandomContent(duration, category)
+      .then(response => {
+        const videoLink = response.data.url
+        setVideoLink(videoLink)
+      })
+      .catch(err => {
```
Comment (sienna-kopf) [inline src/Components/ContentSelection/ContentSelection.js:24]: And error catch! Nice!

File: src/Components/ContentSelection/ContentSelection.js [hunk 2]
```diff
@@ -25,13 +37,14 @@ const ContentSelection = () => {
...
             className={style.button}
             aria-label="somatic exercise"
-            name="somatic exercise"
+            name="SOMATIC"
             onClick={fetchVideo}
           >
             <img
               className={style.somaticIcon}
               src={somaticContentIcon}
               alt="human head with arrows connecting brain to body"
```
Comment (sienna-kopf) [inline src/Components/ContentSelection/ContentSelection.js:46]: Sweet alt tags. And I don't think I said it but @Abdeboskey the icons you drew are AH-mazing 🤩

File: src/apiCalls.js [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
 
 export const updateSettings = (settings) => {
-  return axios.put('https://som-timer-be.herokuapp.com/api/timers/1', {
+  return axios.put(`${rootUrl}/timers/1`, {
     work_interval: settings.workInterval,
     rest_interval: settings.breakInterval
   })
+}
+
+export const getRandomContent = (duration, category) => {
```
Comment (sienna-kopf) [inline src/apiCalls.js:15]: Glad this worked out so easily once we changed the BE to take Query params over a request body.

**Review by sienna-kopf — APPROVED** [review]
MVP 🥳🥳🥳🥳🥳🥳🥳

File: src/apiCalls.js (hunk 3 above)
Comment (Abdeboskey) [reply src/apiCalls.js:15]: Yes, thank you so much for bee-ing so responsive and helpful all the time (to you and all the bees!)

**Review by Abdeboskey — APPROVED** [review]
This looks awesome, it's so great to actually see this delivering content!

File: src/Components/ContentSelection/ContentSelection.js [hunk 4]
```diff
@@ -2,19 +2,31 @@ import React, { useContext} from 'react'
...
     const category = event.target.name
-    //get a video from BE with parameters of category from above & breakInterval duration from settings context 
-    //when get video back, use setVideoLink function to set the video url  
-    //may need a loading screen while this is happening? 
-    setView('content-delivery')
+    const duration = `${settings.breakInterval}:00`
+    
+    getRandomContent(duration, category)
+      .then(response => {
+        const videoLink = response.data.url
```
Comment (rwilliams659) [inline src/Components/ContentSelection/ContentSelection.js:21]: So glad we were able to get this working with the query params!

**Review by rwilliams659 — APPROVED** [review]
Nice work yall! All our endpoints our now hooked up!

**Pull request merged**